src/backends/mysql/session.cpp

```diff
@@ -148,6 +148,23 @@ bool valid_int(const string & s)
     return true;
 }
 
+bool valid_uint(const string & s)
+{
+    char *tail;
+    const char *cstr = s.c_str();
+    errno = 0;
+    unsigned long n = std::strtoul(cstr, &tail, 10);
```

Comment:
`n == 0` if no conversion can be performed. Shouldn't this be taken into account?

Reply:
I will update as follows:

```cpp
    if (errno != 0 || n == 0 || n > UINT_MAX)
```

Is it right?

Reply:
I think I understand correctly. I updated the code.